### PR TITLE
ZEPPELIN-4168: Use secure URLs for Maven repositories

### PR DESCRIPTION
**Pull request by prabhjyotsingh**
### What is this PR for?
Use secure URLs for Maven repositories.

Just run `$ find . -name "pom.xml" | xargs sed -i "" -e "s/http\:/https\:/g"` to make changes in pom.xml and then manually check for some of the references in other files.

### What type of PR is it?
[Bug Fix]

### What is the Jira issue?
* [ZEPPELIN-4168](https://issues.apache.org/jira/browse/ZEPPELIN-4168)

### How should this be tested?
* CI green

### Screenshots (if appropriate)
N/A

### Questions:
* Does the licenses files need update? Yes
* Is there breaking changes for older versions? no
* Does this needs documentation? no

### CODE REVIEW

**Comment by prabhjyotsingh** [comment]
@felixcheung @zjffdu  can you help review this?

**Comment by zjffdu** [comment]
LGTM

File: zeppelin-interpreter/pom.xml [hunk 1]
```diff
@@ -42,7 +42,7 @@
     <maven.plugin.api.version>3.0</maven.plugin.api.version>
     <aether.version>1.12</aether.version>
     <maven.aeither.provider.version>3.0.3</maven.aeither.provider.version>
-    <wagon.version>1.0</wagon.version>
+    <wagon.version>2.7</wagon.version>
```
Comment (felixcheung) [inline zeppelin-interpreter/pom.xml]: is this a required change?

Comment (prabhjyotsingh) [reply zeppelin-interpreter/pom.xml]: Yes, without this CI was failing with 
```
18:26:12,285 ERROR org.apache.zeppelin.spark.DepInterpreter:253 - Exception in DepInterpreter while interpret 
org.sonatype.aether.resolution.DependencyResolutionException: Could not find artifact org.apache.commons:commons-csv:jar:1.1 in central (https://repo.maven.apache.org/maven2/)
	at org.sonatype.aether.impl.internal.DefaultRepositorySystem.resolveDependencies(DefaultRepositorySystem.java:375)
	at org.apache.zeppelin.spark.dep.SparkDependencyContext.fetchArtifactWithDep(SparkDependencyContext.java:171)
	at org.apache.zeppelin.spark.dep.SparkDependencyContext.fetch(SparkDependencyContext.java:121)
	at org.apache.zeppelin.spark.DepInterpreter.interpret(DepInterpreter.java:250)
	at org.apache.zeppelin.spark.DepInterpreterTest.testDefault(DepInterpreterTest.java:78)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
```

With this do I need to update the licence?

**Comment by prabhjyotsingh** [comment]
CI green. Shall I merge this, if no more concerns?

File: zeppelin-interpreter/pom.xml (hunk 1 above)
Comment (felixcheung) [reply zeppelin-interpreter/pom.xml]: probably https://github.com/apache/zeppelin/blob/master/zeppelin-distribution/src/bin_license/LICENSE#L214

**Comment by prabhjyotsingh** [comment]
CI Green (https://travis-ci.org/prabhjyotsingh/zeppelin/builds/537396824) will merge this to master.

**Comment by jongyoul** [comment]
@prabhjyotsingh Hi, I've checked it now but it occurs unknown error on IntelliJ. that IDE might not seem to support https in pom.xml yet. How about waiting for changing the values of "xmlns" and "xmlns:xsi" until that IDE supports it? If there's workaround, could you please share it?

**Comment by prabhjyotsingh** [comment]
Let me give it a try on my intellij, will update this in a while.

**Pull request merged**